api/fixtures/example.go

```diff
@@ -185,6 +185,9 @@ func (o ExampleOptions) Resources() *ExampleResources {
 				},
 			},
 		},
+		Status: hyperv1.HostedClusterStatus{
+			Conditions: []metav1.Condition{},
```

Comment:
Is this actually indicative of a bug in the OpenAPI spec? I notice we're lacking a `omitempty` tag on the `status.conditions` field. If so, maybe you could revert this part and open a new PR for the API fix?

Reply:
@ironcladlou so thats what I tried first actually. We cannot `omitempty` on this field as it will give this error:

```
error: error validating "/Users/krglosse@us.ibm.com/gopath/src/github.ibm.com/alchemy-containers/kodieshandyscripts/render": error validating data: ValidationError(HostedCluster.status): missing required field "conditions" in io.openshift.hypershift.v1alpha1.HostedCluster.status; if you choose to ignore these errors, turn validation off with --validate=false
```

Where status is just simply 
`status:`

Reply:
Would that be indicative of yet another OpenAPI bug? The `.status.conditions` field should be `// +optional` 😁